### ISSUE TITLE
I am requesting the corresponding tutorial for the code. I know it will take some time, but please provide some help to so many people who are interested in your research. Thank you

### ISSUE THREAD

**Issue by tuboshu12** [kolaszko/hl2_detection#3]
Hello, I am a student. I have been researching the code you provided on Github for a long time, but in the end, I couldn't find a way to run it on my HoloLens2, which has caused my current research to stagnate. At the same time, I also see that others are interested in your research but have not found a way to use code. Please provide some help or guidance, and I would be very grateful!!

**Comment by kolaszko** [comment]
Hello @tuboshu12, thanks for a suggestion, I've added an instruction how to run a project in a readme, also made some cleanup in code. In the following days I'll also commit another version of a project, which uses a Sentis library. It makes possible to use a YOLOv5nu, YOLOv6 and YOLOX models, which, according to our tests, perform faster. 
Could you try to follow tutorial to run a project and give a feedback?

**Comment by tuboshu12** [comment]
Of course, it's my honor to work with you. Please provide the versions of software such as Unity and VS. Could you please take some time to provide detailed project operation instructions as much as possible. Because I am a beginner in Unity, thank you very much for your help! I will conduct testing on HoloLens2 glasses based on the project operation instructions you provided and provide timely feedback. In the future, I also want to continue to follow up and make my own improvement plans for the YOLO algorithm mentioned above, and provide them to this project. Thank you again for your reply!

**Comment by tuboshu12** [comment]
> Hello @tuboshu12, thanks for a suggestion, I've added an instruction how to run a project in a readme, also made some cleanup in code. In the following days I'll also commit another version of a project, which uses a Sentis library. It makes possible to use a YOLOv5nu, YOLOv6 and YOLOX models, which, according to our tests, perform faster. Could you try to follow tutorial to run a project and give a feedback?

Hello, I had some problems when running the project. I sent you an email. Could you please help me? Thanks!

**Comment by tuboshu12** [comment]
> Hello @tuboshu12, thanks for a suggestion, I've added an instruction how to run a project in a readme, also made some cleanup in code. In the following days I'll also commit another version of a project, which uses a Sentis library. It makes possible to use a YOLOv5nu, YOLOv6 and YOLOX models, which, according to our tests, perform faster. Could you try to follow tutorial to run a project and give a feedback?

Hello Professor, I have raised some questions about the types and confidence levels of displayed objects. Could you please reply to my email and guide me? Thank you! I've been looking at the code for a long time and still don't understand how to display these two important values.
